### PR TITLE
Updated homepage for v0.3 release

### PR DESCRIPTION
**Pull request by connorchan**
Resolves https://github.com/upbound/www-staging/issues/54
Resolves https://github.com/crossplaneio/crossplaneio.github.io/issues/26

### CODE REVIEW

**Comment by negz** [comment]
@connorchan is this safe to merge?

**Comment by connorchan** [comment]
@negz — not yet. I've just gotten the final designs today. We'll be set to move forward in an hour or two, hopefully

File: index.html [hunk 1]
```diff
@@ -21,237 +21,218 @@ <h1>The open source multicloud control plane.</h1>
       </div>
       <div class="col-7_md-12_md-first hero">
         <div>
-          <img class="img md-hidden" src="{{ "/images/hero.svg" | relative_url }}" />
-          <img class="img md-gt-hidden" src="{{ "/images/hero-mobile.svg" | relative_url }}" />
+          <img class="img" srcset="./images/hero.png 1x, ./images/hero@2x.png 2x, ./images/hero@3x.png 3x" alt="Crossplane hero image" />
```
Comment (rathpc) [inline index.html]: I believe you should still have a `src` along with the `srcset` and all of these should use the `relative_url` helper

File: index.html [hunk 2]
```diff
@@ -21,237 +21,218 @@ <h1>The open source multicloud control plane.</h1>
...
 
-<div id="kinds">
+<div id="use-cases">
   <div>
-    <div class="grid-center">
-      <div class="col-10_sm-12">
-        <h2>The many flavors of managed services we support</h2>
+    <div class="grid">
+      <div class="col-6_md-12 img">
+        <img srcset="./images/feature-1.png 1x, ./images/feature-1@2x.png 2x, ./images/feature-1@3x.png 3x" alt="Provision managed services" />
```
Comment (rathpc) [inline index.html]: I believe you should still have a `src` along with the `srcset` and all of these should use the `relative_url` helper

File: index.html [hunk 3]
```diff
@@ -21,237 +21,218 @@ <h1>The open source multicloud control plane.</h1>
...
+          Add new resources and controllers to Crossplane by extending existing Stacks
+          or building your own.
+          <img src="{{ "/images/quad-red-medium.svg" | relative_url }}" />
+        </p>
       </div>
-      <div class="col-3_md-12 specific-kind">
-        <div class="img"><img src="{{ "/images/database.svg" | relative_url }}" /></div>
-        <h4>Databases</h4>
+      <div class="col-6_md-12_md-first img">
+        <img srcset="./images/feature-4.png 1x, ./images/feature-4@2x.png 2x, ./images/feature-4@3x.png 3x"
```
Comment (rathpc) [inline index.html]: I believe you should still have a `src` along with the `srcset` and all of these should use the `relative_url` helper

File: index.html [hunk 4]
```diff
@@ -21,237 +21,218 @@ <h1>The open source multicloud control plane.</h1>
...
+          alt="Extend Crossplane by adding your own Stack" />
       </div>
-      <div class="col-3_md-12 specific-kind">
-        <div class="img"><img src="{{ "/images/storage.svg" | relative_url }}" /></div>
-        <h4>Storage</h4>
+    </div>
+    <div class="grid">
+      <div class="col-6_md-12 img">
+        <img srcset="./images/feature-2.png 1x, ./images/feature-2@2x.png 2x, ./images/feature-2@3x.png 3x"
```
Comment (rathpc) [inline index.html]: I believe you should still have a `src` along with the `srcset` and all of these should use the `relative_url` helper

File: index.html [hunk 5]
```diff
@@ -21,237 +21,218 @@ <h1>The open source multicloud control plane.</h1>
...
-          <a href="{{githubLink}}"
-            >Help support these features and add new ones</a
-          >
+          Provision and manage kubernetes clusters: bring your own clusters or generate on demand, 
+          with common configuration and policy.
+          <img src="{{ "/images/quad-red-medium.svg" | relative_url }}" />
         </p>
       </div>
+      <div class="col-6_md-12_md-first img">
+        <img srcset="./images/feature-3.png 1x, ./images/feature-3@2x.png 2x, ./images/feature-3@3x.png 3x"
```
Comment (rathpc) [inline index.html]: I believe you should still have a `src` along with the `srcset` and all of these should use the `relative_url` helper

File: index.html [hunk 6]
```diff
@@ -21,237 +21,218 @@ <h1>The open source multicloud control plane.</h1>
...
           ability to orchestrate clouds becomes simple. We look forward to
           collaborating with them on this vision and as the first complex app
           running on Crossplane.&rdquo;
         </p>
         <div class="signature">
-          <div class="name">Sid Sijbrandij</div>
-          <div class="title">CEO and Co-Founder of GitLab</div>
-          <a href="https://gitlab.com"><img src="{{ "/images/gitlab.svg" | relative_url }}"/></a>
+          <div class="exec-img">
+            <img srcset="./images/sid.jpg 1x, ./images/sid@2x.jpg 2x, ./images/sid@3x.jpg 3x" alt="Sid Sijbrandij" />
```
Comment (rathpc) [inline index.html]: I believe you should still have a `src` along with the `srcset` and all of these should use the `relative_url` helper

File: index.html [hunk 7]
```diff
@@ -21,237 +21,218 @@ <h1>The open source multicloud control plane.</h1>
...
+          driven component of the cloud-neutral future.&rdquo;
         </p>
         <div class="signature">
-          <div class="name">Spencer Kimball</div>
-          <div class="title">CEO and Founder of Cockroach Labs</div>
-          <a href="https://www.cockroachlabs.com/"
-            ><img src="{{ "/images/cockroachlabs.svg" | relative_url }}"
-          /></a>
+          <div class="exec-img">
+            <img srcset="./images/spencer.jpg 1x, ./images/spencer@2x.jpg 2x, ./images/spencer@3x.jpg 3x" alt="Spencer Kimball" />
```
Comment (rathpc) [inline index.html]: I believe you should still have a `src` along with the `srcset` and all of these should use the `relative_url` helper

File: index.html [hunk 8]
```diff
@@ -21,237 +21,218 @@ <h1>The open source multicloud control plane.</h1>
...
+          &ldquo;Crossplane has the ability to be at the heart of tomorrow’s multicloud deployments.&rdquo;
+        </h3>
+        <p>
+          &ldquo;It's exciting to see open cloud standards like Crossplane emerge, that will allow new multicloud technologies like
+          YugabyteDB to take advantage of the broader cloud ecosystem, on-premise and in-public cloud. Future work with
+          Crossplane.io and Rook.io is a natural fit with the Kubernetes ecosystem.&rdquo;
+        </p>
+        <div class="signature">
+          <div class="exec-img">
+            <img srcset="./images/karthik.jpg 1x, ./images/karthik@2x.jpg 2x, ./images/karthik@3x.jpg 3x" alt="Karthik Ranganathan" />
```
Comment (rathpc) [inline index.html]: I believe you should still have a `src` along with the `srcset` and all of these should use the `relative_url` helper

File: index.html [hunk 9]
```diff
@@ -264,14 +245,19 @@ <h3>
           cost.&rdquo;
         </p>
         <div class="signature">
-          <div class="name">Bassam Tabbara</div>
-          <div class="title">CEO and Founder of Upbound</div>
-          <a href="https://upbound.io"><img src="{{ "/images/upbound.svg" | relative_url }}"/></a>
+          <div class="exec-img">
+            <img srcset="./images/bassam.jpg 1x, ./images/bassam@2x.jpg 2x, ./images/bassam@3x.jpg 3x" alt="Bassam Tabbara" />
```
Comment (rathpc) [inline index.html]: I believe you should still have a `src` along with the `srcset` and all of these should use the `relative_url` helper

**Pull request merged**